benchmarks/requirements.txt

```diff
@@ -0,0 +1,9 @@
+scikit-learn==1.1.2
```

Comment:
If we have this, we don't need the `environment.yml` file right?

Reply:
When adding the benchmarks to the pre-commits, we could even add the packages to the setup.py?